### DISCUSSION TITLE
Docker ID for rustic

### DISCUSSION THREAD

**Discussion by mueckinger** [rustic-rs/rustic#995] [General]
The Docker ID is the first part of the image name e.g. `docker pull <dockerID>/<repository>`.
rustic is already being used by another user, so `docker pull rustic/rustic` is not possible.
Docker ID must be between 4 and 30 characters long, and can only contain numbers and lowercase letters, so `rustic-rs/rustic` is also not possible. What would be your proposals?
`rusticrs/rustic` or `rusticbackup/rustic` ...

**Comment by simonsan** [comment]
- `rustic-dev/rustic`
- `rustic-for-docker/rustic`
- `rustic.cli.rs/rustic`
- `rustic-maintainers/rustic`

**Reply by mueckinger** [reply]
Sorry, only numbers and lowercase letters - that's tricky ;-)

**Reply by mueckinger** [reply]
* `rusticdocker/rustic`

**Comment by OrbitingOcelot** [comment]
May I suggest, if possible, to consider adding the rclone binary into the container, or proving an alternate image variant with rclone included, as requested here:
https://github.com/rustic-rs/rustic/issues/1008#issuecomment-1900339370
